### DISCUSSION TITLE
How do I jump into a file's path in the search result view?

### DISCUSSION THREAD

**Discussion by wastee** [sxyazi/yazi#804] [1. Q&A] [answered]
If it's a directory, I can press the 'l' key to 'cd' into the path.
But how can I 'cd' into the path of a file?

**Comment by lpnh** [comment]
Hi! If I understand your needs correctly, this [wrapper](https://yazi-rs.github.io/docs/quick-start/#shell-wrapper) could help with that. Exiting Yazi should then bring you to the selected directory.

**Reply by wastee** [reply]
It doesn't seem to be the direct way to achieve my target. However, it's a nice similar way to refer to. Thank you.

**Comment by sxyazi** [comment, marked as answer]
I think implementing the `enter` command for files from search results as well can address this issue, would like to create a feature request for it?

**Reply by wastee** [reply]
~~Yes, absolutely. If it's convenient for you.~~
------
~~I created a FR. https://github.com/sxyazi/yazi/issues/813~~
------
I am such an idiot. Thanks for the help. It's a straightforward way to achieve it.
https://github.com/sxyazi/yazi/issues/813#issuecomment-1994777273
